### ISSUE TITLE
Bug: Docs

### ISSUE THREAD

**Issue by JRMeyer** [coqui-ai/STT#2231]
In STT playbook....`python3 train.py` is mentioned...not.... `python -m coqui_stt_training`

**Comment by abouquet** [comment]
May be related

`train.py`  link in `TRAINING.md` refer to a broken page

```
https://stt.readthedocs.io/en/latest/Flags.html#training-flags
```

**Comment by wasertech** [comment]
Yeah the playbook needs to be updated. See also #2315 which fixes the docs to use docker image version to use `stt-train:latest` instead of `stt-train:v0.10.0-alpha.4`.
